### ISSUE TITLE
[Feature Request] Disable "auto-mount after gathering" for fisher

### ISSUE THREAD

**Issue by nonamethanks** [PunishXIV/PandorasBox#85]
Unlike miner and botanist, Fisher doesn't work by looping through nodes, so there's no reason for this feature to apply to that job.

**Comment by Uriei** [comment]
I disagree, spearfishing exists and works exactly like that, looping through gathering nodes.

**Comment by nonamethanks** [comment]
Well, then detect whether the character was spearfishing or is underwater before mounting up and jumping every time it stops.
